test/equilateral_triangle_test.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:test/test.dart';
+import 'package:flutter_work/oop/shapeHierarchy.dart';
+import 'dart:math';
+
+void main() {
+  group('Test Equilateral Triangle Class', () {
+    test('Perimeter for each side = 3', () {
```

Comment:
A test must be structured in Given, When, Then parts. For example:
// Given trianglr
...
// When compute perimeter
...
// Then value must be the expected

Reply:
Done